src/main/java/uk/gov/ons/census/casesvc/model/dto/ReceiptDTO.java

```diff
@@ -18,6 +18,8 @@
 
   private boolean unreceipt;
 
+  private String agentId;
```

Comment:
How does this relate to the story in question?

Reply:
Discussed this with @NickGrantONS and renaming this to ResponseDTO.